src/utils/Constants.ts

```diff
@@ -46,32 +46,74 @@ export const FORMATS = {
   rpm: 'rpm'
 };
 
+export const PURL_TYPES = {
+  alpine: 'alpine',
+  cargo: 'cargo', //cargo == crates == rust
+  chocolatey: 'chocolatey',
+  clojars: 'clojars',
+  cocoapods: 'cocoapods',
+  composer: 'composer', //packagist website but composer format, php language
+  conan: 'conan',
+  conda: 'conda',
+  cran: 'cran',
+  debian: 'deb',
+  gem: 'gem',
+  github: 'github',
+  golang: 'golang',
+  maven: 'maven',
+  npm: 'npm',
+  nuget: 'nuget',
+  pypi: 'pypi',
+  rpm: 'rpm'
+};
+
 export const REPOS = {
   alpineLinux: 'alpineLinux',
   cratesIo: 'cratesIo',
-}
+  anaconda: 'anaconda',
+  chocolatey: 'chocolatey',
+  clojars: 'clojars',
+  cocoapods: 'cocoapods',
+  conan: 'conan',
+  cran: 'cran',
+  debianpackages: 'debianpackages',
+  debiantracker: 'debiantracker',
+  godev: 'godev',
+  mavenrepo1: 'mavenrepo1',
+  mavenapache: 'mavenapache',
+  mavencentral: 'mavencentral',
+  mvnrepository: 'mvnrepository',
+  npmJS: 'npmjs',
+  nuget: 'nuget',
+  packagist: 'packagist',
+  pypi: 'pypi',
+  rubygems: 'rubygems',
+  nexusepo: 'nexusrepo',
+  artifactory: 'artifactory'
+};
 
 export interface RepoType {
   url: string;
   repoFormat?: string;
   repoID: string;
   titleSelector?: string;
   versionPath?: string;
-  dataSource: string;
+  dataSource: string; //TODO: remove this as unnecessary now as we no longer switch datasources on RepoType
```

Comment:
Yeah we should just get rid of this, I think. I wondered if it was useful to say like a format CAN get data from IQ or OSS Index though. What do you think?

Reply:
Hmm yes, not a bad idea. Then should be a collection of some kind?

Reply:
Likely, we can cross that bridge in a while though. I think it's safe to leave it until then though.

Reply:
Yes agreed for now. We'll leave it. Just a reminder to get to it when we need to.